components/card/stories/card.stories.js

```diff
@@ -39,25 +39,44 @@ export const Basic = Template.bind({});
 Basic.args = {
 	label: 'Basic',
 	heading: 'Title',
-	'supporting-text': 'Supporting Text'
+	'card-text': 'Supporting Text'
 };
 
 export const Subtitle = Template.bind({});
 Subtitle.args = {
 	label: 'Subtitle',
 	heading: 'Title',
 	subtitle: 'Subtitle',
-	'supporting-text': 'Supporting Text'
+	'card-text': 'Supporting Text'
 };
 
 export const TrimmedTitles = Template.bind({});
 TrimmedTitles.args = {
 	label: 'Trimmed titles',
 	heading: 'A Long Title that can get up to an infinite number of rows or you can set the number of rows that will be shown until they are trimmed',
 	subtitle: 'Subtitle that can get up to an infinite number of rows or you can set the number of rows that will be shown until they are trimmed',
-	'supporting-text': 'Supporting Text'
+	'card-text': 'Supporting Text'
 };
 
+const TopActionTemplate = args => html`
+	<style>
+		#root-inner {
+			width: 400px
+		}
+	</style>
+	<vwc-card ...=${spread(args)} header-icon="chat-line">
+		<vwc-icon-button-toggle onicon="more-vertical-solid" officon="more-vertical-solid" slot="meta"></vwc-icon-button-toggle>
```

Comment:
what is `onIcon`?

Reply:
the icon that is when the button is on. I copy paste from` icon-button-toggle` :)